core/aws_ddk_core/stages/dms_s3_to_s3.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from typing import Any, Dict, List, Optional, Union
+
+from aws_cdk.aws_events import EventPattern, IRuleTarget
+from aws_cdk.aws_s3 import IBucket
+from aws_ddk_core.pipelines.stage import DataStage
+from aws_ddk_core.resources import DMSFactory
+from constructs import Construct
+
+
+class DMSS3ToS3Stage(DataStage):
```

Comment:
With support of other integrations besides S3 I'd look into making this an abstract DMS stage that can be configured with different source/target endpoints